browser/ui/views/toolbar/wallet_button_notification_source.cc

```diff
@@ -30,6 +30,12 @@ void WalletButtonNotificationSource::EnsureTxServiceConnected() {
   if (!tx_service_) {
     return;
   }
+
+  // Already connected.
+  if (tx_observer_.is_bound()) {
```

Comment:
nit: move to the top of this function, same as below.

Reply:
Done.